src/instructlab/config/config.py

```diff
@@ -0,0 +1,15 @@
+#from lab import ilab
+import click
+from .init import init
+
+@click.group(chain=True)
```

Comment:
I don't think `chain=True` is appropriate here, but let me know if I'm missing something

Reply:
lets sync about `chain=True` click doesn't like when this option isn't specified (click weirdness we need to move away from in the future!)

Reply:
Can you be a bit more explicit about the click weirdness?